app/controllers/components/course/users_component.rb

```diff
@@ -28,17 +28,28 @@ def main_sidebar_items
     ]
   end
 
+  # Direct the 'Manage Users' link to the usual course_users_students_path if current course user is a manager,
+  # otherwise direct it to manage personalized timelines.
   def admin_sidebar_items
-    return [] unless can?(:manage, CourseUser.new(course: current_course))
+    can_manage_users = can?(:manage, CourseUser.new(course: current_course))
+    can_manage_personal_times =
+      current_course.show_personalized_timeline_features? && can?(:manage_personal_times, current_course)
+
+    return [] unless can_manage_users || can_manage_personal_times
 
     [
       {
         icon: 'user-plus',
         title: t('layouts.course_users.title'),
         type: :admin,
         weight: 1,
-        path: course_users_students_path(current_course),
-        unread: current_course.enrol_requests.count
+        path:
+          if can_manage_users
+            course_users_students_path(current_course)
+          else
+            personal_times_course_users_path(current_course)
+          end,
+        unread: can_manage_users ? current_course.enrol_requests.count : 0
```

Comment:
Actually do I need to do this? Since it seems teaching staff can access enroll requests anyway.